data/types/base/selfmonitored/self_monitored.go

```diff
@@ -9,8 +9,9 @@ import (
 type BloodGlucose struct {
 	base.Base `bson:",inline"`
 
-	Value *float64 `json:"value,omitempty" bson:"value,omitempty"`
-	Units *string  `json:"units,omitempty" bson:"units,omitempty"`
+	Value   *float64 `json:"value,omitempty" bson:"value,omitempty"`
+	Units   *string  `json:"units,omitempty" bson:"units,omitempty"`
+	SubType *string  `json:"subType,omitempty" bson:"subType,omitempty"`
```

Comment:
If this adds a subType, you'll need to add the Meta object (look at basal, bolus, or device for the Meta()).  The base.Meta() only has type, which is why basal, bolus, and device all have their own Meta() function that adds the subType (or deviceType).